dask/array/tests/test_ghost.py

```diff
@@ -1,6 +1,9 @@
-import dask.array as da
+import pytest
```

Comment:
FWIW I've been trying to avoid tying dask to a particular testing solution.  Most of dask works equally well under nose or pytest